docs/developer/dependencies.md

```diff
@@ -59,16 +59,18 @@ sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_heade
 
 ## Haskell Stack
 
-When you're done, ensure `stack --version` is >= 1.6.5
+Please refer to [Stack's installation instructions](https://docs.haskellstack.org/en/stable/README/#how-to-install).
+
+When you're done, ensure `stack --version` is recent, ideally the same as `STACK_ALPINE_VERSION` in [`build/alpine/Dockerfile.prebuilder`](../../build/alpine/Dockerfile.prebuilder).
 
 You may wish to make executables installed by stack available, by e.g. adding the following to your shell profile:
 
 ```bash
 export PATH=~/.local/bin:$PATH
 ```
```

Comment:
Should we also update this?

Our Makefile installs stuff into `dist`, so running `make install` and following this hint will still not make the binaries available.
- Should we change it to `<path_to_project>/dist`?
- Or add a sentence saying that it depends on whether you use the Makefile or plain `stack install`?
- Or leave it out? (`stack install` will by itself already leave a note saying "Warning: Installation path /home/mheinzel/dev/wire-server/dist not found on the PATH environment variable.")

Reply:
I think we could remove that entire paragraph - I never run `brig` globally. If people use stack elsewhere, or use it to install globally-available binaries, then they can add that to their path, but it isn't needed for wire-server development as far as I can see.

Reply:
Thanks, I removed it.